python/tskit/trees.py

```diff
@@ -2433,6 +2434,10 @@ def split_polytomies(
         :param int random_seed: The random seed. If this is None, a random seed will
             be automatically generated. Valid random seeds must be between 1 and
             :math:`2^32 − 1`.
+        :param \\**kwargs: Further arguments used as parameters when constructing the
```

Comment:
I'm not sure this is how **kwargs is best documented. How does this render in HTML? We could just put in a paragraph saying "Additional keyword arguments supplied to this function are passed to the :class:`Tree`` constructor. For example, ...".

Reply:
That's how we've previously done it, e.g. in `aslist`. Happy to swap, but I guess we should be consistent @jeromekelleher ?

Reply:
OK, if we've done it that way before then lets stick with it.